x-pack/plugin/blob-cache/preallocate/src/main/java/org/elasticsearch/blobcache/preallocate/Preallocate.java

```diff
@@ -26,10 +25,18 @@ public class Preallocate {
 
     private static final Logger logger = LogManager.getLogger(Preallocate.class);
 
+    private static final boolean IS_LINUX;
```

Comment:
I'm surprised we don't have our own utility for this in core lib or something.

Reply:
I was equally surprised. We have this in several places within the build as well. For now I just copied this since it was so small, but we may want to consider adding a utility so that not everything has to depend on Lucene directly.